src/components/node-list/node-list-items.test.js

```diff
@@ -97,7 +97,7 @@ describe('node-list-selectors', () => {
 
     it('filters expected number of items', () => {
       expect(filteredTagItems.length).not.toBe(tags.length);
-      expect(filteredTagItems).toHaveLength(2);
+      expect(filteredTagItems).toHaveLength(1);
```

Comment:
Could we use a search with 2 or more results ideally?

Reply:
This is set up to match the results from the searchvalue that was set up in the test - I had modified the search term to generate more filtered search results.